crates/turborepo-analytics/src/lib.rs

```diff
@@ -324,7 +328,7 @@ mod tests {
 
     #[tokio::test]
     async fn test_closing() {
-        let (tx, _) = mpsc::unbounded_channel();
+        let (tx, _rx) = mpsc::unbounded_channel();
```

Comment:
this should be a clippy lint.. got stuck just today not understanding why the daemon was shutting down because I assigned one end of a channel to `_`

Reply:
Yeah, I'm really surprised it's not. Creating a closed channel feels like it's almost never what a developer wants.